tests/test_mod.py

```diff
@@ -11,6 +11,23 @@
 from sentinelsat.sentinel import (InvalidChecksumError, SentinelAPI, SentinelAPIError, convert_timestamp, format_date,
                                   get_coordinates, md5_compare)
 
+_small_query = dict(
+        area='0 0,1 1,0 1,0 0',
+        initial_date=datetime(2015, 1, 1),
+        end_date=datetime(2015, 1, 2))
+
+_large_query = dict(
+        area='0 0,0 10,10 10,10 0,0 0',
+        initial_date=datetime(2015, 1, 1),
+        end_date=datetime(2015, 12, 31))
+
+_api_auth = dict(
```

Comment:
I think it would be a good idea to check if the tests marked with `@pytest.mark.scihub` are enabled and fail fast if the environment variables are not set. I haven't looked into how this might be done but I assume it should not be too difficult.